src/main/asciidoc/concepts/provisioning/provisioning.adoc

```diff
@@ -18,20 +18,20 @@
 //
 === Provisioning
 The main purpose of identity management systems is to manage user and role provisioning.
-User and role provisioning refers to the creation, maintenance, activation and deactivation of user and role objects 
-and their attributes. Provisioning operations can act on Apache Syncope only or be propagated towards external 
+User and role provisioning refers to the creation, maintenance, activation and deactivation of user and role objects
+and their attributes. Provisioning operations can act on Apache Syncope only or be propagated towards external
 resources as well.
-The provisioning operation can be initiated by an authorized user (for instance, working on Apache Syncope 
+The provisioning operation can be initiated by an authorized user (for instance, working on Apache Syncope
 administration console) or by an internal task like a pull task.
-A push task can be used to perform a bulk provisioning operation involving either Syncope and one 
+A push task can be used to perform a bulk provisioning operation involving either Syncope and one
 or more external resources.
 
 include::connectors.adoc[]
 
-include::resources.adoc[]
-
 include::propagation.adoc[]
 
+include::resources.adoc[]
```

Comment:
The document for resources refers to propagation which is described after it. This was a bit confusing when I read it, so I made this change. Should I remove this?